### DISCUSSION TITLE
有什么办法解决同步问题吗

### DISCUSSION THREAD

**Discussion by bingtsingw** [sonofmagic/monorepo-template#42] [General]
我以前是用[mrm](https://github.com/sapegin/mrm)，不过这个库已经不更新了，也不支持monorepo

**Comment by sonofmagic** [comment]
目前这个仓库用的:
https://github.com/sonofmagic/monorepo-template/tree/main/packages/monorepo
来做同步

mrm 的确很久不更新了
